package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "request": "^2.79.0",
     "request-promise": "^4.1.1",
-    "restify": "^4.3.0"
+    "restify": "^4.3.0",
```

Comment:
I think we should use ~ instead of ^. That way, we control when major/minor upgrades occur. A ~ will only allow the revision number to change.